src/renderer/widgets/IWidget.cpp

```diff
@@ -32,9 +32,9 @@ Vector2D IWidget::posFromHVAlign(const Vector2D& viewport, const Vector2D& size,
     else if (valign == "top")
         pos.y += viewport.y - size.y;
     else if (valign == "bottom")
-        pos.y += size.y;
+        pos.y += 0;
     else if (valign != "none")
-        Debug::log(ERR, "IWidget: invalid halign {}", halign);
+        Debug::log(ERR, "IWidget: invalid valign {}", valign);
```

Comment:
I mean this error

Reply:
but you are right, it is actually listed as valid in the wiki xD

Reply:
Uhm I am sorry it is actually `!=`. kinda missed that.
Yeah then you are right @bvr-yr and this is probably intended.

Reply:
but what's wrong here, it's `!= "none"` ?
shouldn't print if `none`

Reply:
yeah if it is `none` it wont print.
I missed that `none` means "don't align" and overlooked that it is a valid value.